src/plugins/adjacent/adjacent.js

```diff
@@ -28,7 +28,19 @@ u.prototype.adjacent = function(html, data, callback) {
 
       return u(part).nodes;
     }).each(function(n){
-      fragment.appendChild(n);
+      var cloneWithEvents = n.cloneNode(true),
```

Comment:
Yeah no problem and thank you for the input that will speed up my solution. Should I be withdrawing pr's while they're still in progress?